packages/scandipwa/src/util/Cart/Token.js

```diff
@@ -13,28 +13,48 @@ import { isSignedIn } from 'Util/Auth';
 import BrowserDatabase from 'Util/BrowserDatabase';
 
 export const GUEST_QUOTE_ID = 'guest_quote_id';
+export const WEBSITE_QUOTE_TOKENS = 'Token.website_quote_tokens';
```

Comment:
1. Why didn't you reuse the `GUEST_QUOTE_ID` variable?
2. Why did you leave it?
3. Why the values don't match the code style (lowercase letters divided by underscore)?

Please simply reuse the `GUEST_QUOTE_ID` value.